docs/querying/api.md

```diff
@@ -256,6 +256,7 @@ URL query parameters:
   series to return. At least one `match[]` argument must be provided.
 - `start=<rfc3339 | unix_timestamp>`: Start timestamp.
 - `end=<rfc3339 | unix_timestamp>`: End timestamp.
+- `limit=<number>`: Maximum number of returned series. Optional.
```

Comment:
Documentation of `limit` should be repeated on the label-names and label-values APIs.